paket.dependencies

```diff
@@ -5,7 +5,6 @@ storage: none
 framework: netstandard2.0, netcoreapp3.0
 
 nuget Expecto
-nuget Microsoft.NET.Sdk.Functions >= 1.0.24
```

Comment:
we can remove this because there's a matching functions sdk ref in each of the netstandard2.0 and netcoreapp3.0 groups.